augmenty/augment_utilities.py

```diff
@@ -42,14 +45,37 @@ def set_doc_level(
     Returns:
         Callable[[Language, Example], Iterator[Example]]: The combined augmenter
     """
+
     def __augment(nlp: Language, example: Example):
         if random.random() > level:
             yield example
         else:
             for e in augmenter(nlp, example):
                 yield e
+
+    return __augment
+
+
+def yield_original(
+    augmenter: Callable[[Language, Example], Iterator[Example]]
+) -> Callable[[Language, Example], Iterator[Example]]:
+    """Wraps and augmented such that it yields both the original and augmented example.
+
+    Args:
+        augmenter (Callable[[Language, Example], Iterator[Example]]): A spaCy augmenters.
+
+    Returns:
+        Callable[[Language, Example], Iterator[Example]]: The augmenter, which now yields both the original and augmented example.
+    """
```

Comment:
if using with `set_doc_level`, make sure it doesn't return the same example twice if the example has not been augmented

Reply:
I have added this